packages/plugin-core/src/commands/pods/BaseExportPodCommand.ts

```diff
@@ -283,10 +283,13 @@ export abstract class BaseExportPodCommand<
               const filteredPayload = opts.payload.filter(
                 (note) => note.fname !== savedNote.fname
               );
-              await this.executeExportNotes({
-                ...opts,
-                payload: filteredPayload.concat(savedNote),
-              });
+              // export only selection if present instead of entire body
+              const exportOpts =
```

Comment:
can we raise an warning if nothing is selected?

Reply:
We don't have a separate scope for `clipboard`. We export the selected text if the export scope is `note` and selection is present. Not very sure if that would be the right call to raise a warning on each note export.

Reply:
can we modify this so we actually create a new scope "selection" and only apply this on selection? 

this way, we can be specific about the selection case. we also don't want to accidentally export the selection when the user meant to export the note.

copy as should then always set the `selection` scope

Reply:
got it

Reply:
[warning toaster](https://github.com/dendronhq/dendron/pull/3619/files#diff-b300d2a5c67b0dd9dd53963dac70b830cd1ae68b29c8de1cc7daeee764cd40bcR409)